### DISCUSSION TITLE
404  on /api/auth-requests

### DISCUSSION THREAD

**Discussion by whoo** [dani-garcia/vaultwarden#3795] [General]
Hello everyone,

I'm encountering an unusual issue after a fresh installation while attempting to enable "device authentication" using the PUSH method. 
```
        - PUSH_ENABLED=true
        - PUSH_INSTALLATION_ID=c*************
        - PUSH_INSTALLATION_KEY=T*************
```

When I click on "connect with another device," the following logs are displayed:

```
vault-vault-1  | [2023-08-24 00:43:48.500][request][INFO] POST /vault/api/auth-requests/
vault-vault-1  | [2023-08-24 00:43:48.500][rocket::server::_][ERROR] No matching routes for POST /vault/api/auth-requests/ application/json.
vault-vault-1  | [2023-08-24 00:43:48.500][rocket::server::_][WARN] Responding with registered (api_not_found) /vault/api 404 catcher.
vault-vault-1  | [2023-08-24 00:43:48.500][response][INFO] 404 Not Found
```

I have followed the configuration for sub-directories: Everything works perfectly (including websocket for password resync), but not for the push functionality. 

Upon startup, all routes are displayed except for this particular one (api/auth-requests).
I definitevely do some-thing wrong.

Best regards,
Whoo

**Comment by stefan0xC** [comment]
What version are you using? This route has been implemented in #3592 which is currently only available in `testing`.

**Reply by Laibalion** [reply]
I was also running into this same issue, not realizing this wasn't in latest image.
Any info on when this gets into latest?

**Comment by whoo** [comment]
Hi,
This is an excellent explanation. 
I simply went into testing... and it works. Thank you.

```
services:
   vault:
     image: vaultwarden/server:testing
     volumes:
        - ./vw-data/:/data
...
```
